client/app/bundles/HelloWorld/components/progress_reports/diagnostic_reports/connect_student_report_box.jsx

```diff
@@ -69,7 +69,7 @@ export default React.createClass({
 										{this.directions(data.directions)}
 										<tr>
 											<td>Prompt</td>
-											<td></td>
+											<td/>
```

Comment:
@emilia-friedberg Do you have a linter that is removing the closing tags in empty cells in the tables? I personally prefer them, but I'll still merge this for now.